src/database/schema/ChatSchema.ts

```diff
@@ -31,6 +31,8 @@ class ChatSchema implements BaseDbSchema {
 
   isContinuous: boolean;
 
+  attachmentJson: any;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachmentJson` property is declared as `any`. This can lead to potential issues with type safety. If possible, define a more specific type or interface for the structure of `attachmentJson`.
```diff
-  attachmentJson: any;
+  attachmentJson: AttachmentType; // Define AttachmentType according to your needs
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->